### PR TITLE
Feature/second best

### PR DESCRIPTION
**Pull request by hroest**
run pyProphet with second best peak group instead of designated decoys
- allows data without decoys
- expands capabilities of pyprophet
- currently tested on the LDA model

### CODE REVIEW

**Comment by hroest** [comment]
@grosenberger @uweschmitt any comments on this, what do you think?

File: pyprophet/config.py [hunk 1]
```diff
@@ -28,6 +28,9 @@ def _standard_config(n_cpus=1):
 
     lambda_ = 0.4
 
+    config["decoy_method"] = "decoy"
+    info["decoy_method"] = r"""[Decoy method to be used (either 'decoy' or 'second_best')]"""
```
Comment (hroest) [inline pyprophet/config.py]: add an additional option to use either decoys or second_best

**Comment by grosenberger** [comment]
👍 Let's merge.

**Comment by uweschmitt** [comment]
I do not see the output files in  the regressions test commit. Did you forget them ?

**Pull request closed without merging**